src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/Router.kt

```diff
@@ -79,20 +84,18 @@ class Router(freeMarkerConfig: Configuration)
     private fun mapNotFound()
     {
         notFound { req, res ->
-            val acceptHeader = req.headers("Accept")
-            if (acceptHeader.contains("text/html") || acceptHeader.contains("*/*"))
+            if (req.pathInfo().startsWith(apiUrlBase))
```

Comment:
Checking the url seems like the most reliable way to decide whether we should return json or a web page.